lms/djangoapps/support/static/support/jsx/errors_list.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+
+class ShowErrors extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidMount() {
+        window.scrollTo(0, 0)
+    }
+
+    render() {
+        return this.props.errorList.length > 0 &&
+                <div className="col-sm-12">
```

Comment:
The indentation here looks weird to me. What is the desired indentation here?

Reply:
it should be 4 spaces, I will update,